byzcoin/bcadmin/main.go

```diff
@@ -399,8 +409,14 @@ func create(c *cli.Context) error {
 		return err
 	}
 
-	fmt.Fprintf(c.App.Writer, "Created ByzCoin with ID %x.\n", cfg.ByzCoinID)
-	fmt.Fprintf(c.App.Writer, "export BC=\"%v\"\n", fn)
+	_, err = fmt.Fprintf(c.App.Writer, "Created ByzCoin with ID %x.\n", cfg.ByzCoinID)
+	if err != nil {
+		return err
```

Comment:
What problem are you trying to solve here? Yes, we know that Printf can fail, but was it really failing? It just seems a little excessive, and wondering how you decided these needed to be checked...

I'm not saying to remove these checks, just curious...

Reply:
Just following IntelliJ's yellow lines...